pandas/io/parquet.py

```diff
@@ -116,13 +117,32 @@ def write(
                 **kwargs,
             )
 
-    def read(self, path, columns=None, **kwargs):
+    def read(self, path, columns=None, use_nullable_dtypes=False, **kwargs):
```

Comment:
how about use_extension_dtypes as more descriptive

Reply:
> how about use_extension_dtypes as more descriptive

It doesn't use extension dtypes in general, only those types that use `pd.NA`

Reply:
See also https://github.com/pandas-dev/pandas/issues/29752 for some discussion about naming this

Reply:
FWIW I would also prefer `use_extension_dtypes`